docs/en/engines/table-engines/special/distributed.md

```diff
@@ -61,19 +61,18 @@ Clusters are set like this:
 ``` xml
 <remote_servers>
     <logs>
-        <shard>
-            <!-- Inter-server per-cluster secret for Distributed queries
-                 default: no secret (no authentication will be performed)
-
-                 If set, then Distributed queries will be validated on shards, so at least:
-                 - such cluster should exist on the shard,
-                 - such cluster should have the same secret.
+        <!-- Inter-server per-cluster secret for Distributed queries
```

Comment:
Seems like the previous version was correct. We have such examples in tests https://github.com/ClickHouse/ClickHouse/blob/persistent_nukeeper_snapshot_storage/tests/integration/test_distributed_inter_server_secret/configs/remote_servers.xml#L15.

Reply:
@alesapin 

Alright, do we need fix this https://clickhouse.tech/docs/en/engines/table-engines/special/distributed/   in official website for distributed engine.

Reply:
Both feature and documentation look correct. `<secret>` tag is a part of shard definition. So I don't understand where is the error?

Reply:
@alesapin  

https://github.com/ClickHouse/ClickHouse/blob/master/src/Interpreters/Cluster.cpp#L307

It's under cluster and all the replicas will share this secret.

Reply:
Yes, you right. Sorry for the misunderstanding.